### ISSUE TITLE
Failed to load settings

### ISSUE THREAD

**Issue by dankmolot** [microsoft/terminal#14827]
### Windows Terminal version

1.16.10262.0

### Windows build number

10.0.22621.0

### Other Software

git version 2.39.1.windows.1
Visual Studio Code 1.75.1 (user setup)
Github desktop Version 3.1.6 (x64)

### Steps to reproduce

I don't know what exactly happened. Day before I have fresh installed windows 11

I suspect that after installing git for windows installer tried to add git bash profile to windows terminal. Day after I have discovered that windows terminal can't load settings, throws obscure error and terminal stats using temporary configuration.

I have tried to delete settings.json and state.json, but nothing happened.

Also in settings.json I haven't seen git bash profile, so maybe git have added profile to somewhere else?
Even reinstalling windows terminal didn't help.

### Expected Behavior

Everything works fine.

### Actual Behavior

Windows Terminal on every startup open this dialog:
![image](https://user-images.githubusercontent.com/44505837/218258393-b4b67e51-ab04-42bb-824c-137c8cbb7e0b.png)

**Comment by dankmolot** [comment]
Terminal launched as administrator doesn't have this issue

**Comment by dankmolot** [comment]
Windows Terminal Preview share the same issue

**Comment by zadjii-msft** [comment]
Huh. Interesting. So, presumably, you've already cleared out your [settings.json file](https://github.com/microsoft/terminal/wiki/Frequently-Asked-Questions-(FAQ)#where-can-i-find-the-settings-file), so sharing the contents of that wouldn't be helpful. 

I'd be curious if one of `state.json`, `settings.json`, or the JSON file that git bash added to the Terminal might be messing with the JSON loading. Alas, I don't think I know where the git bash file would be on disk. Maybe under `C:\ProgramData\Microsoft\Windows Terminal\Fragments\{app-name}\{file-name}.json` or `C:\Users\<user>\AppData\Local\Microsoft\Windows Terminal\Fragments\{app-name}\{file-name}.json`?

If those files look okay, then maybe it's the permissions? Especially if you say that running the Terminal as admin works. Maybe one of those files was written to disk in such a way that the Terminal can't read it as a normal user. If you could share the output of `icacls <json file>`, maybe that might help identify the issue/?

**Comment by dankmolot** [comment]
Thank you for your reply!

**Comment by dankmolot** [comment]
I have found file `git-bash.json` under path `C:\ProgramData\Microsoft\Windows Terminal\Fragments\Git`.
icacls response:
```
.\git-bash.json BUILTIN\Administrators:(I)(F)
                DANIEL-PC\dankm:(I)(F)

Successfully processed 1 files; Failed processing 0 files
```

Also before checking file I have opened it with vscode, and the issue seems to gone. I have opened it before today, and issue still was present, but it is like vscode fixed utf8 file encoding or something...

Thank you for your help! I really wouldn't like to reinstall windows 11 again.
